vulnfeeds/cmd/pypi/main.go

```diff
@@ -147,6 +148,11 @@ func main() {
 			continue
 		}
 
+		if len(notes) > 0 && *withoutNotes {
+			log.Printf("Skipping %s as there are notes associated with it.", vulnPath)
```

Comment:
nit: slightly confusing message, we are skipping since flag is set and not because notes are associated with.
maybe just
Skipping notes in %s ?
or Skipping notes in %s as skip notes flag is set.

Reply:
So a "note" means that a human has to look at it because automation ran into some problems.

This is saying that we are completely skipping all matched vulnerabilities with these notes, so that the only ones that are output are ones without problems. These we can safely auto-commit into the vulnerability database. 

I think the messages you are proposing are saying something different :)

Reply:
Skip, i didnt know what note meant, maybe add a comment in code to explain this.